servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -38,27 +40,36 @@ image:kiali-login.png[Login Page]
 +
 Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
 
+[[kiali-overview-page]]
```

Comment:
I'm not a big fan of screenshots because of the extra maintenance and translation overhead. Does including an image of the overview page provide value to the customer?